plugin/src/main/groovy/com/novoda/staticanalysis/StaticAnalysisPlugin.groovy

```diff
@@ -20,10 +20,11 @@ class StaticAnalysisPlugin implements Plugin<Project> {
     }
 
     private EvaluateViolationsTask createEvaluateViolationsTask(Project project) {
-        StaticAnalysisExtension extension = project.extensions.create('staticAnalysis', StaticAnalysisExtension)
+        StaticAnalysisExtension extension = project.extensions.create('staticAnalysis', StaticAnalysisExtension, project)
```

Comment:
No offense! I found it odd that you're referencing a project `project.extensions.create(...` and then need to pass in the project itself again for inner workings. Dunno, seems repetitive. But what do I know. 😉